bytecounterprocessor/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+type: bytecounter
```

Comment:
It is likely we'll use this processor to count other metrics in the future, such as "event count". Should we make the processor name more generic?

Reply:
Maybe `usageprocessor`?